.azure-pipelines/gpu-tests.yml

```
@@ -18,6 +18,14 @@ pr:
 
 jobs:
   - job: pytest
+    strategy:
+      matrix:
+        'PyTorch - LTS':
+          pyVersion: '3.7'
+          ptVersion: '1.8'
+        'PyTorch - stable':
+          pyVersion: '3.9'
+          ptVersion: '1.11'
```

Sometimes (e.g. when docker images are somehow broken or when unblocking master isn't trivial and needs time), we might want to pin docker images by specifying the hash value, so we could maybe directly specify them here instead of specifying them via `pyVersion` and `ptVersion`.

```suggestion
      matrix:
        'PyTorch - LTS':
          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.8"
        'PyTorch - stable':
          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.11"
```